pkg/security/ebpf/c/defs.h

```diff
@@ -303,6 +303,44 @@ static __attribute__((always_inline)) u32 is_flushing_discarders(void) {
     return prev_id != NULL && *prev_id;
 }
 
+// STATS_MAX_CPU_COUNT represents the maximum number of CPUs that the perf buffer monitoring will sample. Reduce this number
+// to monitor less CPUs and therefore reduce the in-kernel overhead.
+#define STATS_MAX_CPU_COUNT 64
+
+struct perf_map_stats_t {
+    u64 bytes;
+    u64 count;
+    u64 lost;
+};
+
+#define send_event_and_stats(ctx, kernel_event, perf_map, stats_buffer_fb, stats_buffer_bb)                            \
```

Comment:
We're using a macro because the `kernel_event`s provided to the macro have different and non compatible types. The only common ground is the `kernel_event.event` structure, but we still need `kernel_event` to send over the perf map. Hope that makes sense :)